plugins/modules/system/facter.py

```diff
@@ -12,20 +12,38 @@
 module: facter
 short_description: Runs the discovery program I(facter) on the remote system
 description:
-- Runs the I(facter) discovery program
-  (U(https://github.com/puppetlabs/facter)) on the remote system, returning
-  JSON data that can be useful for inventory purposes.
+    - Runs the I(facter) discovery program
+      (U(https://github.com/puppetlabs/facter)) on the remote system, returning
+      JSON data that can be useful for inventory purposes.
```

Comment:
```suggestion
    - Runs the I(facter) discovery program
      (U(https://github.com/puppetlabs/facter)) on the remote system, returning
      JSON data that can be useful for inventory purposes.
```
BTW, the current format is ok as well

Reply:
Shouldn't it be `C(facter)`, not `I(facter)`?